try {
  console.log(this);
}
catch (error) {
  console.error(error);
}